### PR TITLE
CB-12519: added plugins release blog post

### PR DESCRIPTION
**Pull request by stevengill**
plugins release blog post. Please review.

### CODE REVIEW

File: www/_posts/2017-03-06-plugins-release.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+* cordova-plugin-splashscreen@4.0.2
+* cordova-plugin-statusbar@2.2.2
+* cordova-plugin-vibration@2.1.4
+* cordova-plugin-whitelist@1.3.2
+* cordova-plugin-wkwebview-engine@1.1.2
+* cordova-plugin-test-framework@1.1.5
+
+Release Highlights:
+
+* **CAMERA:** [CB-12005](https://issues.apache.org/jira/browse/CB-12005) Changing the `getOrientation` method to return the defined enumerated `EXIF` instead of orientation in degrees for Consistency
```
Comment (dpogue) [inline www/_posts/2017-03-06-plugins-release.md]: **CAMERA** -> **Camera**
Consistency -> consistency

File: www/_posts/2017-03-06-plugins-release.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+* **In App Browser:** [CB-9148](https://issues.apache.org/jira/browse/CB-9148) **Android:** Add Support for `input[type=file]` File Chooser
+* **MEDIA:** Make the output file of **Android** an `acc` file. Major version bump to `3.0.0`. Media plugin now requires `cordova-android >= 6.1.0`
```
Comment (dpogue) [inline www/_posts/2017-03-06-plugins-release.md]: **MEDIA** -> **Media**

File: www/_posts/2017-03-06-plugins-release.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+* **WK Webview Engine:** [CB-12297](https://issues.apache.org/jira/browse/CB-12297) Support `WKProcessPool` for cookie sharing
```
Comment (dpogue) [inline www/_posts/2017-03-06-plugins-release.md]: **WK Webview** -> **WKWebView**

File: www/_posts/2017-03-06-plugins-release.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+* cordova-plugin-statusbar@2.2.2
+* cordova-plugin-vibration@2.1.4
+* cordova-plugin-whitelist@1.3.2
+* cordova-plugin-wkwebview-engine@1.1.2
+* cordova-plugin-test-framework@1.1.5
+
+Release Highlights:
+
+* **CAMERA:** [CB-12005](https://issues.apache.org/jira/browse/CB-12005) Changing the `getOrientation` method to return the defined enumerated `EXIF` instead of orientation in degrees for Consistency
+* **In App Browser:** [CB-9148](https://issues.apache.org/jira/browse/CB-9148) **Android:** Add Support for `input[type=file]` File Chooser
```
Comment (dpogue) [inline www/_posts/2017-03-06-plugins-release.md]: **In App Browser** -> **In-App Browser**

File: www/_posts/2017-03-06-plugins-release.md [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+<!--more-->
+
+cordova-plugin-battery-status@1.2.3
+* [CB-12369](https://issues.apache.org/jira/browse/CB-12369) Add plugin typings from `DefinitelyTyped`
+* [CB-12363](https://issues.apache.org/jira/browse/CB-12363) Added build badges for **iOS 9.3** and **iOS 10.0**
+
+cordova-plugin-camera@2.4.0
+* [CB-12501](https://issues.apache.org/jira/browse/CB-12501) **Android**: Appium tests don't use `XPath` selectors anymore
+* [CB-12469](https://issues.apache.org/jira/browse/CB-12469) Appium tests can now run on **iOS 10**
+* [CB-12005](https://issues.apache.org/jira/browse/CB-12005) Changing the `getOrientation` method to return the defined enumerated `EXIF` instead of orientation in degrees for Consistency
```
Comment (dpogue) [inline www/_posts/2017-03-06-plugins-release.md]: Consistency -> consistency

File: www/_posts/2017-03-06-plugins-release.md [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+* [CB-12369](https://issues.apache.org/jira/browse/CB-12369) Add plugin typings from `DefinitelyTyped`
+* [CB-12363](https://issues.apache.org/jira/browse/CB-12363) Added build badges for **iOS 9.3** and **iOS 10.0**
+* [CB-12230](https://issues.apache.org/jira/browse/CB-12230) Removed **Windows 8.1** build badges
+
+cordova-plugin-whitelist@1.3.2
+* [CB-12236](https://issues.apache.org/jira/browse/CB-12236) Fixed `RELEASENOTES` for `cordova-plugin-whitelist`
+
+cordova-plugin-wkwebview-engine@1.1.2
+* [CB-12497](https://issues.apache.org/jira/browse/CB-12497) `location.href` links are silently disallowed
+* [CB-12490](https://issues.apache.org/jira/browse/CB-12490) - Updated experimental plugin link
```
Comment (dpogue) [inline www/_posts/2017-03-06-plugins-release.md]: Remove - after JIRA number

File: www/_posts/2017-03-06-plugins-release.md [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+* Allow to configure navigation by gestures
+* [CB-12297](https://issues.apache.org/jira/browse/CB-12297) Support `WKProcessPool` for cookie sharing
+* [CB-12414](https://issues.apache.org/jira/browse/CB-12414) **iOS:** Forward error from provisional load error to standard load error
+
+cordova-plugin-test-framework@1.1.5
+* [CB-12236](https://issues.apache.org/jira/browse/CB-12236) - Fixed `RELEASENOTES` for `cordova-plugin-test-framework`
```
Comment (dpogue) [inline www/_posts/2017-03-06-plugins-release.md]: Remove - after JIRA number

File: www/_posts/2017-03-06-plugins-release.md (hunk 4 above)
Comment (filmaj) [inline www/_posts/2017-03-06-plugins-release.md]: Hmm, in the [inappbrowser plugin's release notes, we refer to it as `InAppBrowser`](https://github.com/apache/cordova-plugin-inappbrowser/blob/master/RELEASENOTES.md#170-feb-28-2017). Which should be used?

Comment (dpogue) [inline www/_posts/2017-03-06-plugins-release.md]: docs.cordova.io says `Inappbrowser` (which is definitely wrong IMO), but I guess `InAppBrowser` is probably the most consistent

**Pull request merged**